src/mmda/parsers/grobid_augment_existing_document_parser.py

```diff
@@ -63,12 +64,15 @@ def parse(self, input_pdf_path: str, doc: Document, xml_out_dir: Optional[str] =
     def _parse_xml_onto_doc(self, xml: str, doc: Document) -> Document:
         xml_root = et.fromstring(xml)
 
-        bib_entries = self._get_grobid_bib_box_groups(xml_root)
-        
+        bib_entry_box_groups = self._get_grobid_bib_box_groups(xml_root)
+
+        # get nice SpanGroups from tokens whose centers overlap with the Grobid box
+        bib_entry_span_groups = box_groups_to_span_groups(bib_entry_box_groups, doc, center=True)
```

Comment:
No padding for grobid?

Reply:
I started w/ padding and the test failed, because it grabbed a "]". I think because grobid boxes are so tight around tokens already there's no need for the padding. Grobid bibs don't include the numbers like "[1]" so including the "]" was weird (so I took off the padding)